plugins/inputs/docker_log/docker_log.go

```diff
@@ -198,9 +198,9 @@ func (d *DockerLogs) matchedContainerName(names []string) string {
 func (d *DockerLogs) Gather(acc telegraf.Accumulator) error {
 	ctx := context.Background()
```

Comment:
We need two contexts: this one is for short requests that will end before Gather returns (info and inspect), and another one on 218 for the long running request that tails the log and is cancelled in Stop()

Reply:
Right, this one just gets reset on line 201. For uniformity with line 218, it seems that line 201 should be `context.WithTimeout(context.Background(),`

Reply:
The reason I used this differing style is that, in the future, we will pass in a context to the various plugin functions, and since this first one will essentially be this we can easily just remove the line.  I guess I could have used context.TODO() here, but I don't know, its ugly.

Reply:
Aaah yes, that makes sense.